forte/data/data_store.py

```diff
@@ -1212,11 +1215,40 @@ def co_iterator_annotation_like(
 
         Args:
             type_names: a list of string type names
+            range_annotation: a list with two elements,
```

Comment:
After reading the code, `range_annotation` is not a list of two elements, but the raw Annotation entry data, right? It is fine to pass in the begin and end directly to this function.

Reply:
Yes so `range_annotation` is just a tuple of two ints. I can pass in the two values separately in the `co_iterator_annotation_like` function

Reply:
So a tuple of two `int` feels the same as two `int` parameters to me. But my original comment is "whether the input is the raw Annotation data", which is a list of more than two `int`s, but also include other attributes an Annotation has.

The name `range_annotation` implies that it is an `Annotation` object, which is not only a tuple containing the begin and end.

Reply:
Oh yes I understand. So `range_annotation` is just a `tuple` of two `int`. If you want we can make the name something more understandable.

Reply:
sure, feel free to do so, you can convert this back to a tuple if you want. I am probably confused previously since some function arguments type annotations are also not ready yet.